FILES.deploy-tools/root/playbooks/ndslabs-k8.yml

```diff
@@ -104,12 +104,25 @@
   tasks:
     - os_floating_ip: server="{{ ansible_host }}" 
 
+# Make kubectl work local
+# After this you can use  {{ kubectl }} create -f <...>
+- name: Get master kube config and register kubectl with the configuration
+  hosts: master1
+  roles:
+    - k8s-local-kubectl
+
 # Apply Kubernetes node labels
 - name: NDSLabs initial labels
   hosts: nodes
   roles:
     - ndslabs-k8-init-labels
 
+# Add the cluster-local registry, mapped to all hosts on localhost:5000
+- name: cluster-local-docker-registry
```

Comment:
You are assuming kube-system namespace in the yaml files, so this needs to be created after the addons (which create the namespace). I question whether this belongs in kube-system